pkg/provider/aws/plugin.go

```diff
@@ -11,6 +11,62 @@ import (
 	"go.uber.org/zap"
 )
 
+// ExpandConstructs looks at all existing constructs in the construct graph and turns them into their respective AWS Resources
+func (a *AWS) ExpandConstructs(result *core.ConstructGraph, dag *core.ResourceGraph) (err error) {
+	log := zap.S()
+	constructIds, err := result.TopologicalSort()
```

Comment:
Can you topological sort on a graph that can contain cycles? Does order matter for the expanding?

Reply:
you cant sort on a graph that contains cycles, it will error out, so i will need to remove this. It shouldnt matter since the expansion should be independent of eachother and if they collide in resources, the resources Create method should handle that